### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by orc886**
Warning for equals plugin

### CODE REVIEW

**Comment by SimonCropp** [comment]
a warning without context is not great. can u qualify that warning? is there an issue/doco in the equals project u can link to?

**Comment by orc886** [comment]
I have no idea why adding a comment to the wiki page shows this error

**Comment by SimonCropp** [comment]
ignore the build error. i would like to see some more context as to why "(WARNING: This plugin doesn't work well in the same assembly)"

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ The name "Fody" comes from the small birds that belong to the weaver family [Plo
   * [EmptyConstructor](https://github.com/Fody/EmptyConstructor) Adds an empty constructor to classes even if a non empty one is defined.
   * [EmptyStringGuard](https://github.com/thirkcircus/EmptyStringGuard) Adds empty string argument checks to an assembly.
   * [EnableFaking](https://github.com/philippdolder/EnableFaking.Fody) Allows faking your types without writing interfaces for testing purposes only.
-  * [Equals](https://github.com/Fody/Equals) Generate Equals, GetHashCode and operators methods from properties.
+  * [Equals](https://github.com/Fody/Equals) Generate Equals, GetHashCode and operators methods from properties. (WARNING: This plugin doesn't work well in the same assemnly)
```
Comment (SimonCropp) [inline README.md]: typo: assemnly

**Comment by distantcam** [comment]
I really don't like this PR. The warning should be in the readme of the Equals project instead. Also there's not enough info in this statement to determine if the problem would cause issues for the person using the project.

**Comment by SimonCropp** [comment]
yeah have to agree with @distantcam here. can u add the warning to the equals project

**Comment by orc886** [comment]
It caused problems for me and others who commented on the relevant issue on the Equals project. The issue is unresolved for a long time, and might not be possible to fix. Equals is bad PR for Fody, and if there is no way to implement a simple Equals weaver, it's best if it will be removed. But I guess it's up to you to decide.

**Comment by SimonCropp** [comment]
Can u point to the issue?

**Comment by orc886** [comment]
https://github.com/Fody/Equals/issues/10

**Pull request closed without merging**